django_netjsonconfig/tasks.py

```diff
@@ -0,0 +1,30 @@
+from __future__ import absolute_import, unicode_literals
+
+import requests
+from celery import shared_task
+from django.urls import reverse
+from six.moves.urllib.parse import urlparse
+
+
+@shared_task
+def subscribe(template_id, import_url, subscriber_url, subscribe=False):
+    """
+    Send a subscription notification to template designer
+    """
+    data = {
+        'template': template_id,
+        'subscribe': subscribe,
+        'subscriber': subscriber_url
+    }
+    parse_url = urlparse(import_url)
+    url = '{uri.scheme}://{uri.netloc}'.format(uri=parse_url)
+    path = '{0}{1}'.format(url, reverse('api:notify_template'))
+    requests.post(path, data=data)
+
+
+def base_sync_template_content(subscriber_url, template_id):
```

Comment:
is this a celery task or not? Why is it missing the decorator?

If it's not a task but a helper function, prepend a `_` to the name of the function

Reply:
You could also move this to `TemplateSubscription` as a `staticmethod`